.github/workflows/ci.yml

```diff
@@ -112,6 +89,7 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           interval: 45
           timeout: 1200
+          ref: ${{ github.event.pull_request.head.sha || github.event.push.head.sha }}
```

Comment:
I wonder what is the purpose of this option? 🤔

Reply:
merge gatekeeper is failing in main branch

Reply:
https://github.com/percona/pmm/runs/6908254035?check_suite_focus=true